deploy_helm.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Added color constants
+RED='\033[0;31m'
+NC='\033[0m'
+
+# Add announcement that this script has just started with timestamp
+echo -e "${RED}Script started at $(date +%T)${NC}"
+
+# Create dev environment if not exist
+echo -e "${RED}Creating dev environment${NC}"
+kubectl create namespace dev
+
+# Create staging environment if not exist
+echo -e "${RED}Creating staging environment${NC}"
+kubectl create namespace stag
+
+# Create production environment if not exist
+echo -e "${RED}Creating production environment${NC}"
+kubectl create namespace prod
+
+# Read the argument with selected namespace
+ns="${1:-"dev"}"
```

Comment:
This doesn't respect your local setup if you want to you namespace from your kubeconfig context and specify only chart name.

Better option is to use getopt() and replace parameters witch arguments e.g., -e env_name -n chart_name.